libp2p/NodeTable.cpp

```diff
@@ -32,6 +33,30 @@ namespace
 BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_discoveryWarnLogger,
     boost::log::sources::severity_channel_logger_mt<>,
     (boost::log::keywords::severity = 0)(boost::log::keywords::channel = "discov"))
+
+struct DatagramTypeTableEntry
+{
+    char const* name;
+    uint8_t type;
+};
+
+DatagramTypeTableEntry datagramTypesTable[] = {
+    {"PING", PingNode::type},
+    {"PONG", Pong::type},
+    {"FINDNODE", FindNode::type},
+    {"NEIGHBORS", Neighbours::type}
+};
+
+string datagramNameFromType(uint8_t _t)
```

Comment:
I'd suggest adding `typeName()` method to `RLPXDatagramFace` (and each implementation returning its name) instead of this. Then you won't need to handle unknown packet case and no need to throw.

Reply:
Good idea, I'll go ahead and do that